mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td

```diff
@@ -810,7 +837,7 @@ def Tosa_MulOp : Tosa_ElementwiseOp<"mul", [
   let arguments = (ins
     Tosa_Tensor:$input1,
     Tosa_Tensor:$input2,
-    I8Attr:$shift
+    TosaTensorRankOf<[Tosa_Int8], [1]>:$shift
```

Comment:
Why not make this optional rather than having to drag over zero shift tensors and parameters around.

Reply:
Good point, done.